apps/ui/src/components/Ui/TokenStamp.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
```

Comment:
I think it should be called `StampToken` based on [naming convention](https://vuejs.org/style-guide/rules-strongly-recommended#order-of-words-in-component-names).

@samuveth

Reply:
Yes, `StampToken`